src/injected/client-init.ts

```diff
@@ -17,3 +38,5 @@ if (!window.windowInitializer) {
 
     window.windowInitializer.initialize().catch(logger.error);
```

Comment:
Now that this is in an async context, probably preferable to just await it and move the `.catch(logger.error)` up a level to L42